.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
```

Comment:
Any reason why we need to test so many node versions? Shouldn't the node version of the Docker image be enough?